### PR TITLE
Add Chinese Translation (2023.4.4) 

### PR DESCRIPTION
**Pull request by SleepyAsh0191**
Some things need pay attention to:
* I can't determine the word about Apep(new weekly boss)

### CODE REVIEW

File: dataset/dictionary/characters.json5 [hunk 1]
```diff
@@ -2854,24 +2860,29 @@
...
   {
     en: "Apep",
     ja: "アペプ",
+    zhCN: "阿佩普",
     tags: [ "sumeru", "character-sub", "enemy-boss" ],
   },
   {
     en: "Dragon of Verdure",
     ja: "翠の草龍",
+    zhCN: "草龙(暂定)",   //TODO 中文版译名暂缺，“草龙”是俗称。
```
Comment (xicri) [inline dataset/dictionary/characters.json5:2885]: Let's comment out this Chinese translation until the accurate traslation is revealed.

**Review by xicri — CHANGES_REQUESTED** [review]
Thanks!
I added some comments.

File: dataset/dictionary/characters.json5 [hunk 2]
```diff
@@ -2854,24 +2860,29 @@
...
+    zhCN: "斯露莎",
     tags: [ "sumeru", "character-sub" ],
   },
   // TODO Because I'm not sure Apep / アペプ is the name of the new weekly boss,
   // I may update this word after v3.6 has released
+  // (new) アペプのオアシス守護者 is in Preview page of v3.6
   {
     en: "Apep",
     ja: "アペプ",
+    zhCN: "阿佩普",
```
Comment (xicri) [inline dataset/dictionary/characters.json5:2879]: > I can't determine the word about Apep(new weekly boss)

Can you ellaborate what you mean?

I checked the Simplified Chinese version of the v3.6 preview page, and 阿佩普 looks OK for the Chinese translation of Apep / アペプ.

I'm not sure if Apep / アペプ / 阿佩普 should be listed in the dictionary as a boss name, so I will determine after v3.6 release.
Guardian of Apep's Oasis / アペプのオアシス守護者 / 阿佩普的绿洲守望者 might be the correct boss name, and if so, I will change this translation.

**Comment by xicri** [comment]
@Bill-Haku Can you review this PR as always?

File: dataset/dictionary/characters.json5 (hunk 2 above)
Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:2879]: Apep seems to be a name of the place, maybe we cannot get the name of the boss until the new version is released.

**Review by Bill-Haku — APPROVED** [review]

File: dataset/dictionary/characters.json5 (hunk 1 above)
Comment (xicri) [reply dataset/dictionary/characters.json5]: Fixed in 810c233 and bb0a4d7.

**Review by xicri — APPROVED** [review]
@SleepyAsh0191 Thanks, looks good!

File: dataset/dictionary/characters.json5 [hunk 3]
```diff
@@ -2854,24 +2860,29 @@
...
     tags: [ "sumeru", "character-sub", "enemy-boss" ],
   },
   {
     en: "Dragon of Verdure",
     ja: "翠の草龍",
+    // zhCN: "草龙(暂定)", TODO 中文版译名暂缺，“草龙”是俗称。
```
Comment (xicri) [inline dataset/dictionary/characters.json5:2885]: I found it.
Maybe 翠草之龙?

zh-CN
en
ja

Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:2885]: That seems correct

Comment (xicri) [reply dataset/dictionary/characters.json5:2885]: @Bill-Haku Thanks, I'll apply by myself.
cc: @SleepyAsh0191

Comment (xicri) [reply dataset/dictionary/characters.json5:2885]: I just noticed these screenshots might be a spoiler... Apologies if you saw something you don't want to see yet.

Comment (Bill-Haku) [reply dataset/dictionary/characters.json5:2885]: It's okay. I don't mind it.

**Pull request merged**